elevator.js

```diff
@@ -136,6 +136,14 @@ var Elevator = function(options) {
         }
     };
 
+	this.setVerticalPadding = function(vp) {
+		verticalPadding = vp;
+	};
+
+	this.isElevating = function() {
```

Comment:
Can you turn above variable line:118 to isElevating and its usage instead creating this function.